src/shared/components/donate/donate.js

```diff
@@ -1,32 +1,32 @@
-import React from 'react';
-// import PropTypes from 'prop-types';
+import React, { Component } from 'react';
 import Section from 'shared/components/section/section';
 import LinkButton from 'shared/components/linkButton/linkButton';
 import styles from './donate.css';
```

Comment:
`styles` was already used as a class name. does this muddy up the global scope? I'm not sure with the eslint-plugin-react-with-styles.